site/public/css/server.css

```diff
@@ -244,6 +244,14 @@ button,
 /****************************
  * TABLE STYLES
  ***************************/
+
+@media (prefers-color-scheme: dark) {
```

Comment:
This solution is not preferred, because there is a settings page where the user can specify that they want the system's preferred color scheme. This looks like it will do the check again and may override the user's selection on their profile page. Another possible solution would be to just add the `table-striped` class to the tables on the page which will do the checking of which color scheme the user wants for you.

Reply:
Only I have to add table-striped class in table, right??

And "the tables on **the page which will do the checking of** which color scheme the user wants for you" I am confused with this sentence. You mean NumericRubric.twig file you are talking about, right??

Reply:
I am facing these kind of problem as I am unable to see changes and effects in API because I could not instal submitty locally to check API's operations because of issue on my computer.

Reply:
> Only I have to add table-striped class in table, right??
> 
> And "the tables on **the page which will do the checking of** which color scheme the user wants for you" I am confused with this sentence. You mean NumericRubric.twig file you are talking about, right??

Yup, though the class was actually `table-stripe`, my mistake. And turns out the problem was slightly more complicated than I had thought. Currently, the rows get cloned when we try to create more rows. However, the count of the row also gets cloned and each row ends up having an "even" count and thus the background color is not properly alternated. Instead of cloning the row, we can instead just insert a row using javascript.